.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
+
+# The Auro team will be the default owners for everything in the repo.
+*       @AlaskaAirlines/generalauroreviewers
+
+# Order is important. The last matching pattern has the most precedence.
+# If a pull request touches any files in the ./src dir, only these owners
+# will be requested to review.
+./src/*    @AlaskaAirlines/auroteamreviewers
```

Comment:
Add additional return to the end of all files as described in the editor's config.

Reply:
Hi @blackfalcon ,

I will update this file as your feedback thank you.